pkg/webhook/validator.go

```diff
@@ -494,13 +507,11 @@ func ValidatePolicySignaturesForAuthority(ctx context.Context, ref name.Referenc
 		// https://github.com/sigstore/policy-controller/issues/1652
 		sps, err := valid(ctx, ref, rekorClient, authority.Key.PublicKeys, remoteOpts...)
 		if err != nil {
-			return nil, fmt.Errorf("failed to validate public keys with authority %s for %s: %w", name, ref.Name(), err)
-		} else if len(sps) > 0 {
-			logging.FromContext(ctx).Debugf("validated signature for %s with authority %s got %d signatures", ref.Name(), authority.Name, len(sps))
-			return ociSignatureToPolicySignature(ctx, sps), nil
+			return nil, fmt.Errorf("key validation failed for authority %s for %s: %w", name, ref.Name(), err)
 		}
-		logging.FromContext(ctx).Errorf("no validSignatures found with authority %s for %s", name, ref.Name())
-		return nil, fmt.Errorf("no valid signatures found with authority %s for %s", name, ref.Name())
```

Comment:
As I said above, this is unreachable today.

Reply:
yes, it makes sense.